### PR TITLE
Node: Solana watcher cleanup

### PR DESCRIPTION
**Pull request by bruce-riley**
This PR does some simple clean up / performance improvements in the Solana watcher.

### CODE REVIEW

**Review by panoel — COMMENTED** [review]
Don't know if there is only 1 account key matching the contract or if you want the last one?  :shrug:

**Comment by bruce-riley** [comment]
> Don't know if there is only 1 account key matching the contract or if you want the last one? 🤷

That is a question that I asked @evan-gray offline. If we are uncomfortable with this change, I can take it out.

**Review by bingyuyap — APPROVED** [review]
lgtm assuming you want the 1st account key you found

**Comment by bruce-riley** [comment]
I removed the break that was causing some concern.

**Review by bingyuyap — APPROVED** [review]

File: node/pkg/watchers/solana/client.go [hunk 1]
```diff
@@ -215,21 +222,27 @@ func NewSolanaWatcher(
 	queryReqC <-chan *query.PerChainQueryInternal,
 	queryResponseC chan<- *query.PerChainQueryResponseInternal,
 ) *SolanaWatcher {
+	msgObservedLogLevel := zapcore.DebugLevel
+	if chainID != vaa.ChainIDPythNet {
```
Comment (evan-gray) [inline node/pkg/watchers/solana/client.go:226]: this did confuse me the first time I read it. might be easier to read to have the default be info and then if it *is* pythnet, set it to debug.

**Review by panoel — APPROVED** [review]

**Review by evan-gray — APPROVED** [review]

**Pull request merged**